templates/emails/com/welcome.html

```diff
@@ -13,7 +13,7 @@ <h1 style="margin-top: 0; color: #333333; font-size: 22px; font-weight: bold; te
 {% block content %}
   {{ render_text("My name is Son. I’m the founder of SimpleLogin and I wanted to be the first to welcome you on board.") }}
 
-  {{ render_text('To better secure your account, I recommend enabling Multi-Factor Authentication (MFA) on your <a href="https://app.simplelogin.io/dashboard/setting/#totp">Setting page</a>.') }}
+  {{ render_text('To better secure your account, I recommend enabling Multi-Factor Authentication (MFA) on your <a href="' ~ URL ~ '/dashboard/setting/#totp">Setting page</a>.') }}
```

Comment:
The `render_text` is slowly replaced by the `{% call text() %}` which is much nicer and avoids the escape-hell :), an example is here:
https://github.com/simple-login/app/blob/9ddb8ff2d4134516255e9eca86bfb250d5f395d2/templates/emails/transactional/spf-fail.html#L12

I wonder if we should take the opportunity to use the `{% call text() %}` here?

Reply:
I agree - `{% call text() %}` is better. But there are many more code parts which have to be updated. I will do that in another pull request (as soon as this one is merged) to not overload this one.